desktop/mac/privileged-helper.md

```diff
@@ -0,0 +1,39 @@
+---
+description: Docker Desktop Privileged Helper on Mac
+keywords: Docker Desktop, mac, security, install
+title: Docker Desktop Privileged Helper on Mac
+redirect_from:
+- /docker-for-mac/privileged-helper/
+---
+
+## Permission requirements
+
+While Docker Desktop does not generally require the user running or installing it to have `root` privileges, in the default use case it needs root access to be granted on the first run. The first time Docker Desktop is launched the user gets an admin prompt to grant permissions for a privileged helper service to be installed. For subsequent runs no `root` privileges are required. 
+
+The reason for this is that Docker Desktop needs to perform a limited set of privileged operations which is done by the privileged helper process `com.docker.vmnetd`. This approach allows, following the principle of least privilege, `root` access to be used only for the operations for which it is absolutely necessary while still being able to use Docker Desktop as an unprivileged user.
+
+From version 4.11 it will be possible to avoid running a privileged service in the background by using `com.docker.vmnetd` for setup during installation and disabling it at runtime. In this case the user will not be prompted on the first run. Administrators would be able to do that by using the `–user` flag on the [install command](install.md#install-from-the-command-line) which would:
```

Comment:
```suggestion
From version 4.11, it will be possible to avoid running a privileged service in the background by using `com.docker.vmnetd` for setup during installation and disabling it at runtime. In this case the user will not be prompted on the first run. Administrators would be able to do that by using the `–user` flag on the [install command](install.md#install-from-the-command-line) which would:
```